src/content/platform-integration/android/install-android/index.md

```diff
@@ -1,8 +1,8 @@
 ---
-title: Add Android devtools for Flutter
+title: Add Android as a target platform for Flutter
 description: Configure your system to develop Flutter for Android.
-short-title: Add Android DevTools
-target-list: [Windows, 'Web on Windows', Linux, 'Web on Linux', macOS, 'Web on macOS', iOS, Web on ChromeOS]
+short-title: Add Android compiling
```

Comment:
Change to either "Set up Android compilation" or "Configure Android compilation"